docs/migrations/v1_to_v2.md

```diff
@@ -0,0 +1,65 @@
+- [Obsolete APIs have been removed](https://github.com/DataDog/dd-trace-dotnet/tree/v2.0.1/docs/Datadog.Trace#obsolete-apis-have-been-removed)
+- [Introduction of interfaces `ISpan`, `IScope`, and `ITracer`](https://github.com/DataDog/dd-trace-dotnet/tree/v2.0.1/docs/Datadog.Trace#elasticsearchnet5-integration-id-removed)
+- [Simplification of the tracer interface](https://github.com/DataDog/dd-trace-dotnet/tree/v2.0.1/docs/Datadog.Trace#simplification-of-the-tracer-interface)
+- [Incorrect integration names are ignored](https://github.com/DataDog/dd-trace-dotnet/tree/v2.0.1/docs/Datadog.Trace#incorrect-integration-names-are-ignored)
+- [Automatic instrumentation changes](https://github.com/DataDog/dd-trace-dotnet/tree/v2.0.1/docs/Datadog.Trace#automatic-instrumentation-changes)
+
+### What if you are relying on .NET Framework lower than 4.6.1
+
+If you are currently targeting version < 4.6.1, we suggest you upgrade in line with [Microsoft's guidance](https://docs.microsoft.com/en-us/lifecycle/products/microsoft-net-framework). On our end, we will support fixing major bugs on top of Tracer v1.31.
+
+That said, automatic instrumentation relies only on the CLR (runtime) version, not the compile-time target. Even if an application is built for .NET Framework older than 4.6.1, as long as it runs on a supported CLR version, automatic instrumentation will work normally.
+
+Regarding custom instrumentation, NuGet package version 2.0 and above will no longer include binaries for versions of .NET Framework older than 4.6.1. The NuGet package will only work correctly with applications that target (at compile time):
+
+- .NET Framework 4.6.1 or above
+- .NET Core 3.1 or above
+- .NET 5 or above
+- .NET Standard 2.0 or above (includes .NET Core 2.0 and above)
+
+### Rollout strategy when using automatic and custom instrumentation
+
+If your application uses both automatic and custom instrumentation, your application traces may be disconnected until both components are upgraded to v2.x. Follow the steps below to upgrade the components in the recommended order.
+
+**Note:** To minimize application overhead, align the versions of the automatic and custom instrumentation. Using two different versions of the 2.x .NET Tracer will still result in complete traces but it requires additional overhead, so Datadog recommends minimizing the amount of time that the versions are mismatched.
+
+#### .NET Core and .NET 5+
+
+##### If you were using v1.28.8 or greater
+
+On .NET Core, to avoid disconnected traces during the upgrade process:
+
+1. Upgrade the NuGet package to the latest version of the tracer
+2. Upgrade the automatic instrumentation package to the matching version of the tracer
+
+> If you are targeting .NET Core/.NET 5+, and are unable to upgrade both the _Datadog.Trace_ NuGet package and MSI simultaneously, always prefer upgrading the NuGet package first.
```

Comment:
I added a comment in the "rollout strategy" section above that could make this comment redundant. What do you think of the suggestion to move it up?

Reply:
🤔, I don't see exactly why. I think this comment works only in v1.28.8 and above section. But we can discuss offline and I can rearrange obv.

Reply:
Discussed offline. Thanks